### ISSUE TITLE
Audience joined callback

### ISSUE THREAD

**Issue by EpariNigam** [AgoraIO/Basic-Video-Broadcasting#68]
How can I get, an audience joined the session?

**Comment by plutoless** [comment]
there's no such callback for the moment.

**Comment by plutoless** [comment]
yuo may use our rtm sdk to manage member join/leave events

**Comment by EpariNigam** [comment]
In that case, I will have separate user id in rtc and rtm right?

**Comment by plutoless** [comment]
right. you might need to manage the mappings yourself in your backend
